recipes/restinio/v0.7/conanfile.py

```diff
@@ -37,12 +37,12 @@ def layout(self):
     def requirements(self):
         self.requires("llhttp/9.1.3")
         self.requires("fmt/10.2.1")
-        self.requires("expected-lite/0.6.3")
+        self.requires("expected-lite/0.8.0")
```

Comment:
This will create a conflict with jinja2cpp, still using previous version. Bump deps should be better orchestrated with others or wait until/if they can be moved to version-ranges

Reply:
This is still conflicting with jinja2cpp

Reply:
@memsharded, as I am now also adding `0.7.3` and that version requires these, I am adding a conditional check on the version..

Hope this will allow us merging this. thx!